src/derivation.js

```diff
@@ -81,6 +81,9 @@ const modes = Object.freeze({
       TEZOS_XPUB_CURVE: "SECP256K1" // FIXME bug in libcore? it should be ED25519
     }
   },
+  tron: {
+    overridesDerivation: "44'/195'/<account>'/0/0"
```

Comment:
i don't think it's required because it's basically the default derivation no? (bip44)

Reply:
Yes it is, I thought it was mandatory field.